datasync/fetcher/doc.go

```diff
@@ -18,5 +18,15 @@
 // This file is derived from eth/fetcher/fetcher.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package fetcher contains the block announcement based synchronisation.
+/*
+Package fetcher contains the block announcement based synchronisation.
+ How fetcher works
+ If the node receives a whole block, fetcher inserts the block into the chain and broadcast it to its peers.
+If a block hash is received instead of a block, node requests and reflects the header and body from the peer who sent the block hash.
```

Comment:
Above `if` has a space before itself, however the below one doesn't. Could you please check?

Reply:
There is no space character. I thought the first sentence needed a space. It is not.